### DISCUSSION TITLE
programmatically configure the width of the Command Log

### DISCUSSION THREAD

**Discussion by guillaumecle** [cypress-io/cypress#29402] [General]
I'm already able to configure the viewport and the window size (leveraging this https://www.cypress.io/blog/2021/03/01/generate-high-resolution-videos-and-screenshots) but the Command Log (also known as reporter) is always using the default width of 450 (from `runner-constants.ts`).

Is it possible to programmatically it? I know that I can manually drag it but I want to do it automatically for CI.

**Comment by guillaumecle** [comment]
I could use this trick but it's hacky and shift the whole content
```
const reporterPanel = top!.document.querySelector("[data-cy='reporter-panel']");
(reporterPanel as HTMLElement).style.width = '1000px';
```

**Reply by BillyTom** [reply]
For posterity, this is the content of `runner-constants.ts`:

```
export const runnerConstants = {
  defaultSpecListWidth: 280,
  defaultReporterWidth: 450,
  preferredMinimumPanelWidth: 200,
  absoluteAutMinimum: 100,
  absoluteSpecListMinimum: 50,
  absoluteReporterMinimum: 50,
  collapsedNavBarWidth: 64,
}
```

I also would like to increase the default reporter width. I create screenshots in a very wide resolution, and the reporter width should be increased proportionally.

I tried to use the hack that you described, but it shifts the whole content as you said.
